rest-endpoint/src/main/java/com/cognifide/aet/rest/MetadataServlet.java

```diff
@@ -192,6 +190,21 @@ private void checkLock(String suiteIdentifier) throws AETException {
     }
   }
 
+  private void createNotFoundResponse(HttpServletResponse response, String correlationId, String suiteName, String suiteVersion, DBKey dbKey) throws IOException {
+    response.setStatus(HttpURLConnection.HTTP_NOT_FOUND);
+    String paramsValuesMessage = "";
+    if (correlationId != null) {
+      paramsValuesMessage = String.format("correlationId: %s", correlationId);
+    } else if (suiteName != null) {
+      paramsValuesMessage = String.format("suite name: %s", suiteName);
+      if (suiteVersion != null) {
+        paramsValuesMessage = String.format("%s, version: %s", paramsValuesMessage, suiteVersion);
```

Comment:
Is everything ok with formatting?

Reply:
Looks fine to me, these are nested if statements. It's OK according to Google Style